pretty.js

```diff
@@ -47,6 +47,7 @@ function filter (value) {
 }
 
 function isPinoLine (line) {
+  if (line === null) return false
   return line.hasOwnProperty('hostname') && line.hasOwnProperty('pid') && (line.hasOwnProperty('v') && line.v === 1)
```

Comment:
I think we can go multiline with `return line &&`, if `line` is falsy there is no chance for it to have the other properties.